internal/backend/remote-state/s3/backend.go

```diff
@@ -1026,14 +1010,6 @@ func (b *Backend) Configure(obj cty.Value) tfdiags.Diagnostics {
 		Token:                   stringAttr(obj, "token"),
 	}
 
-	// The "legacy" authentication workflow used in aws-sdk-go-base V1 will be
-	// gradually phased out over several Terraform minor versions:
-	//
-	// 1.6 - Default to `true` (prefer existing behavior, "opt-out" for new behavior)
-	// 1.7 - Default to `false` (prefer new behavior, "opt-in" for legacy behavior)
-	// 1.8 - Remove argument, legacy workflow no longer supported
-	cfg.UseLegacyWorkflow = boolAttr(obj, "use_legacy_workflow")
```

Comment:
Can we now remove `UseLegacyWorkflow` from https://github.com/hashicorp/aws-sdk-go-base?

Reply:
I believe so as I don't think we ever exposed it in the AWS provider. I'll open an issue.

Reply:
Opened https://github.com/hashicorp/aws-sdk-go-base/issues/946